engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/UpsertVisualizationJob.scala

```diff
@@ -302,13 +312,19 @@ object UpsertVisualizationJob {
     * @return either the evaluation result or an evaluation error
     */
   private def evaluateVisualizationExpression(
+    module: String,
     expression: Api.VisualizationExpression
   )(implicit
     ctx: RuntimeContext
   ): Either[EvaluationFailure, EvaluationResult] = {
     for {
-      module     <- findModule(expression.module)
-      expression <- evaluateModuleExpression(module, expression)
+      module           <- findModule(module)
+      expressionModule <- findModule(expression.module)
+      expression <- evaluateModuleExpression(
+        module,
+        expression,
+        expressionModule
```

Comment:
Why do we need both `module` and `expressionModule`?

Reply:
One is used for `prepareFunctionCall` and the other one for `evaluateExpression`.